### ISSUE TITLE
Allow a way to give an output folder to -enable-clang-static-analyzer

### ISSUE THREAD

**Issue by pocc** [oclint/oclint#537]
### Issue Summary
Reopening #330 as it is still a problem

### Environment
bash-5.0$ oclint --version
LLVM (http://llvm.org/):
  LLVM version 5.0.0svn-r313528
  Optimized build.
  Default target: x86_64-apple-darwin18.5.0
  Host CPU: haswell

OCLint (http://oclint.org/):
  OCLint version 0.13.
  Built Sep 18 2017 (08:58:40).

oclint installed via homebrew

### Reproduction Steps
`oclint -enable-clang-static-analyzer /path/to/file.c` will save a plist named `file.plist` in the current directory. This can be partially avoided by using `cd /tmp; <oclint command above>; cd -`, but this is kludgy.

### Expected Behavior
Expected behavior is to not create detritus in working directories (or use /tmp or similar for such purposes).

### Actual Behavior
plist file is saved in a working directory

**Comment by ryuichi-assistant** [comment]
There hasn't been any activity on this issue recently. Due to the high number of incoming GitHub notifications, we have to clean some of the old issues, as many of them have already been resolved with the latest updates.

Please make sure to update to the latest version and check if that solves the issue. Let us know if that works for you by adding a comment.

**Comment by pocc** [comment]
I have verified that this is still an issue on v20.11 for Ubuntu 20.20. This is the temp.c that I am using

    int main() {int i;}

With command `./oclint -enable-clang-static-analyzer temp.c`

This is the generated temp.plist in the same directory

```html
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
 <key>clang_version</key>
<string>clang version 11.0.0 (https://github.com/llvm/llvm-project.git 0160ad802e899c2922bc9b29564080c22eb0908c)</string>
 <key>diagnostics</key>
 <array>
 </array>
 <key>files</key>
 <array>
 </array>
</dict>
</plist>
```

**Comment by ryuichi-assistant** [comment]
There hasn't been any activity on this issue recently. Due to the high number of incoming GitHub notifications, we have to clean some of the old issues, as many of them have already been resolved with the latest updates.

Please make sure to update to the latest version and check if that solves the issue. Let us know if that works for you by adding a comment.

**Comment by pocc** [comment]
As far as I'm aware, this is still an issue.
